examples/dial-psk/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/pion/dtls"
+	"github.com/pion/dtls/examples/util"
+)
+
+func main() {
+	// Prepare the IP to connect to
+	addr := &net.UDPAddr{IP: net.ParseIP("127.0.0.1"), Port: 4444}
+
+	//
+	// Everything below is the pion-DTLS API! Thanks for using it ❤️.
+	//
+
+	// Prepare the configuration of the DTLS connection
+	config := &dtls.Config{
+		PSK: func(hint []byte) ([]byte, error) {
```

Comment:
Since `hint` isn't used it might be nice to use the blank, `_`, identifier instead to make it more obvious to the reader?

Reply:
Oh yea good point, what do you think of https://github.com/pion/dtls/pull/76/commits/0a9c0ced8977b98ce9037943e11b4d089f696ca0

The `hint` is used, and we print it to try and help give the user a better understanding of its purpose.

Reply:
That works too! :+1: